src/ArbitrumFoundationVestingWallet.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {IERC20VotesUpgradeable} from "./Util.sol";
+
+interface IL2ArbitrumGoverner {
+    function token() external view returns (IERC20VotesUpgradeable);
+    function EXCLUDE_ADDRESS() external view returns (address);
+}
```

Comment:
seems like this interface is declared twice in the codebase. would be good to have a single one that is imported both here and in the gov actions contracts
https://github.com/ArbitrumFoundation/governance/blob/50d3cefeb75af95bedae896f867431b2d98050de/src/gov-action-contracts/address-registries/L2AddressRegistryInterfaces.sol#L32-L41

Reply:
yeah the interface situation is a bit unlovely, might just move all the contract interfaces in L2AddressRegistryInterfaces to their own src level dir